distribution/docker/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM knotx/knotx:2.0.0
```

Comment:
I think we should use 2.0.1-SNAPSHOT here, such as `knotx.version` from  `distribution/docker/gradle.properties`.

Reply:
Or we can switch to `2.0`...

Reply:
I switched to `latest`, is that ok?